projects/cashmere/src/lib/scroll-nav/scroll-nav.module.ts

```diff
@@ -10,7 +10,7 @@ import {ScrollNavLinkDirective} from './nav/scroll-nav-link.directive';
 @NgModule({
     declarations: [HcScrollNavContentComponent, HcScrollNavComponent, ScrollNavTargetDirective, ScrollNavLinkDirective],
     exports: [HcScrollNavContentComponent, HcScrollNavComponent, ScrollNavTargetDirective, ScrollNavLinkDirective],
-    imports: [CommonModule, ScrollDispatchModule],
+    imports: [CommonModule, ScrollingModule],
```

Comment:
Is there a certain version or version range of the CDK that this change will work with?

Reply:
@joeskeen  ScrollDispatchModule was deprecated in version 7, and we're requiring cashmere implementers to be on angular 8